packages/nimbus/src/components/tooltip/tooltip.mdx

```diff
@@ -13,36 +11,119 @@ menu:
+- **Position tooltips appropriately:** Place tooltips so they don't obscure the element they're related to or other important UI elements. Consider the user's mouse path and avoid accidental triggers.
+- **Maintain accessibility:** Includes keyboard navigation and screen reader compatibility. Use ARIA attributes like `aria-describedby`to link the tooltip to its target element.
+- **Avoid overuse:** Don't use tooltips for every single UI element. Overuse can make the interface feel cluttered.
+- **Don't rely on tooltips for essential information:**  Crucial information (like error states) should never be exclusively conveyed through tooltips, as they are not always discoverable or accessible to all users. Use tooltips for supplementary or secondary details.
 
-[React Aria Components Tooltip Documentation](https://react-spectrum.adobe.com/react-aria/Tooltip.html)
-[Tooltip Aria Pattern](https://www.w3.org/TR/wai-aria-1.2/#tooltip)
 
-## Basic Usage
+## Usage
 
-All instances of `Tooltip` consist of two components, the `TooltipTrigger` and
-the `Tooltip`.
+Tooltips are a valuable tool for providing concise, contextual information, but they should be used strategically and not as a replacement for clear and accessible design. Think carefully about the type of information you're presenting and choose the most appropriate UI element for the job.
 
-The `TooltipTrigger` is required to wrap both the trigger element and the
-`Tooltip`.
+
+### Using tooltips for icons buttons
+
+Icons often rely on shared universal understanding. A tooltip provides a text label that removes any ambiguity and ensures users understand the button's function. This is especially important for less common or custom icons.
+
+> [!TIP]\
+> **Do**
+>
+> - Describe the icon’s action quickly and concisely.  
+
+```jsx-live
+const App = () => <Flex height="100px">
+  <img src="/images/tooltip/tooltip-guideline-do.png" alt="tooltip do guideline" />
+</Flex>
+```
+
+> [!CAUTION]\
+> **Don't**
+>
+> - Do not describe the icon visually.
+> - Do not over explain the action to the user.
+> - Don’t overlap tooltips with the object it is clarifying.
 
 ```jsx-live
-const App = () => (
-  <TooltipTrigger>
-    <Button>Trigger Element</Button>
-    <Tooltip>Tooltip</Tooltip>
-  </TooltipTrigger>
-  )
+const App = () => <Flex height="100px">
+  <img src="/images/tooltip/tooltip-guideline-dont.png" alt="tooltip do guideline" />
+</Flex>
 ```
 
-## Tooltip Trigger Props
 
-<PropTable id="TooltipTrigger" />
 
-## Tooltip Props
+### Specs
 
 <PropTable id="Tooltip" />
+
+### Accessibility
+
+Tooltip triggers must be focusable and hoverable in order to ensure that all users can activate them. When displayed, TooltipTrigger automatically associates the tooltip with the trigger element so that it is described by the tooltip content.
+
+Authors SHOULD ensure that elements with the `role` tooltip are referenced through the use of `aria-describedby` before or at the time the tooltip is displayed.
```

Comment:
I think the texts, `role` and `aria-describedby`, in the figma file suggests a link to the mozilla a11y docs for each case. 

```suggestion
Authors SHOULD ensure that elements with the [role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles) tooltip are referenced through the use of [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby) before or at the time the tooltip is displayed.
```